C/ChibiScheme/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+using BinaryBuilder
+
+name = "ChibiScheme"
+version = v"0.10"
+
+# Collection of sources required to build NLopt
+sources = [
+    GitSource("https://github.com/ashinn/chibi-scheme.git",
+              "05eb4ebd357e2bf2fe5aa7da13b0422ce20ddf7f"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/chibi-scheme
+mkdir build && cd build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    ..
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(; experimental=true)
```

Comment:
`experimental=true` isn't necessary anymore
```suggestion
platforms = supported_platforms()
```